spec/mongoid/fields_spec.rb

```diff
@@ -425,6 +425,17 @@
       end
     end
 
+    context "when the Symbol type is used" do
+
+      before do
+        expect(Mongoid.logger).to receive(:warn)
+      end
+
+      it "warns that the BSON symbol type is deprecated" do
+        Band.field :should_warn, type: Symbol
+      end
+    end
+
     context "when the options are not valid" do
```

Comment:
Let's make sure we delete the field off the ``Band`` model after this test.
Unlike the test right above this one, the field is actually added so some cleanup is necessary.

for example:
```ruby
        after do
          Band.fields.delete("acceptable")
        end
```